lib/blockscore/base.rb

```diff
@@ -53,21 +54,25 @@ def self.resource
       @resource ||= Util.to_underscore(to_s.split('::').last)
     end
 
-    def self.api_url
-      'https://api.blockscore.com/'
-    end
-
     def self.endpoint
       fail NotImplementedError, ABSTRACT_WARNING if equal?(Base)
 
-      "#{api_url}#{Util.to_plural(resource)}"
+      Pathname(Util.to_plural(resource))
+    end
+
+    def member_endpoint
+      endpoint + id
```

Comment:
I just wanted to confirm, because I didn't see any specs, but does this properly add a slash in between the `endpoint` and `id`? I looked at the implementation of the `endpoint` class method and I didn't get any clues whether this could be the case.